File: hello.f
      program hello
      implicit none
      integer i
      write(*,'(a)') "hello world"
      write(*,'(a)') "fortan"
<<<<<<< HEAD
      do i=1,3,1             !print beeps at end of program
=======
      do 10 i=1,3,1             !print beeps at end of program
>>>>>>> 2b2867668869afb8408043162da19f340886786d
         if(i.eq.1) then
            write(*,'(i2,2(a))')i," beep", char(7)
         else
            write(*,'(i2,2(a))')i," beeps", char(7)
         endif
         call sleep(1)
<<<<<<< HEAD
      enddo
=======
 10   continue
>>>>>>> 2b2867668869afb8408043162da19f340886786d
      end 

File: hello.c
#include <stdio.h> //required for printf
<<<<<<< HEAD
#include <unistd.h> //required for sleep
=======
//#include <unistd.h> //required for sleep
>>>>>>> 2b2867668869afb8408043162da19f340886786d
int main()
{
  printf("hello world\n");
  printf("C++\n");
  for(int i=1;i<4;i++) {//print beeps at end of program
    printf(" %d beeps\a\n",i);
<<<<<<< HEAD
    sleep(1);
=======
    //sleep(1);
    system("sleep 1");
>>>>>>> 2b2867668869afb8408043162da19f340886786d
  }
  return 0;
}
